api/src/main/resources/db/migration/1.0/V1.0.65__DML-INSERT_DATA-PROGRAM_ALGORITHM_RULE.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO PROGRAM_ALGORITHM_RULE (PROGRAM_ALGO_RULE_ID,GRADUATION_PROGRAM_CODE,ALGORITHM_RULE_CODE,SORT_ORDER) VALUES ('EEE0819D9426A0FEE0539AE9228EF8F7','2018-PF','EQUIVALENCY_RULE',95);
```

Comment:
Hi @infstar, to simplify, you can use SYS_GUID() instead of hard-coding values for IDs. For example: INSERT INTO PROGRAM_ALGORITHM_RULE (PROGRAM_ALGO_RULE_ID,GRADUATION_PROGRAM_CODE,ALGORITHM_RULE_CODE,SORT_ORDER) VALUES (SYS_GUID(),'2018-PF','EQUIVALENCY_RULE',95);

Reply:
I talked with Sreepad or Kamal before that we want to use the same GUID for different environments for the important codes related to Grad Algorithm.
That's why SYS_GUID() is not used in the flyway script.